proto/zepben/protobuf/hc/opendss/Diagnostics.proto

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2020 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "com.zepben.protobuf.hc.opendss";
+option csharp_namespace = "Zepben.Protobuf.HC.OpenDSS";
+
+//
+// NOTE: We have deliberately removed the package to prevent name mangling in c where these classes are used. The
+//       other languages we use do not reference the package, so will still be correctly scoped.
+//
+
+/**
+ * Message for streaming the summary report
+ */
+
+message SummaryReport {
+    int64 timestamp = 1;
+    string circuitName = 2;
```

Comment:
Not specifically, I just believe it's easier to understand the meaning of it when it's `circuitName` rather than `CaseName`. However, I don't have a strong preference, I can restore it to CaseName.

Reply:
is it always a circuit? i assume there's some reason they called it CaseName in OpenDSS?

Reply:
It pulls the value from the `CaseName` field of the active circuit rather than the `Name` field, so I assume you can change this to be different to the circuit name if you want to